crates/usvg/src/parser/converter.rs

```diff
@@ -596,6 +608,7 @@ pub(crate) fn convert_group(
     force: bool,
     cache: &mut Cache,
     parent: &mut Group,
+    is_context_element: bool,
```

Comment:
Since this flag doesn't affect `convert_group` logic, let's set it to `false` by default. And modify in `use_node.rs` when necessary. 

`convert_group` already has too many arguments and because Rust doesn't support named arguments - it looks like a mess.